quartz/plugins/transformers/lastmod.ts

```diff
@@ -49,10 +49,8 @@ export const CreatedModifiedDate: QuartzTransformerPlugin<Partial<Options>> = (u
                 modified ||= st.mtimeMs
               } else if (source === "frontmatter" && file.data.frontmatter) {
                 created ||= file.data.frontmatter.date as MaybeDate
```

Comment:
aliased as `frontmatter.created` in `frontmatter.ts` but referred to as `frontmatter.date` here